wiki/History_of_osu!/ru.md

```diff
@@ -0,0 +1,29 @@
+---
+outdated: true
+needs_cleanup: true
+---
+
```

Comment:
```suggestion
```

Reply:
`outdated` should stay, we recently changed how these tags work

Reply:
> `outdated` should stay, we recently changed how these tags work

what about `needs_cleanup`? need to delete?

Reply:
yeah `needs_cleanup` should be removed

Reply:
all done.